File: mayur/HomeAppliancesSpring/src/main/java/com/example/demo/entities/Appliance.java
package com.example.demo.entities;


import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Table(name="appliances")
public class Appliance {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	int App_id;
	
	@Column
	String name;
	
	@Column
	String description;
	
	@Column
	byte[] image;
	
	@Column
	Double pricepermonth;
	
	@ManyToOne
	@JsonIgnoreProperties("appliances")
	@JoinColumn(name="user_id")
	User user;
	
	
	@ManyToOne
	@JsonIgnoreProperties("appliances")
	@JoinColumn(name="cat_id")
	Category category;
	
	@Column
	int isverified;
	
	@Column
	int onrent;


	public Appliance(String name, String description, double pricepermonth,
			User user, Category category, int isverified,int onrent) {
		this.name = name;
		this.description = description;
		this.pricepermonth = pricepermonth;
		this.user = user;
		this.category = category;
		this.isverified = isverified;
		this.onrent = onrent;
	}

}

File: Main Project/Backend .net/RentIT/Models/Appliance.cs
using System;
using System.Collections.Generic;

namespace RentIT.Models
{
    public partial class Appliance
    {
        public int AppId { get; set; }
        public string? Description { get; set; }
        public byte[]? Image { get; set; }
        public string? Name { get; set; }
        public double? Pricepermonth { get; set; }
        public int? CatId { get; set; }
        public int? UserId { get; set; }
        public int? Isverified { get; set; }
        public int? Onrent { get; set; }

        public virtual Category? Cat { get; set; }
        public virtual User? User { get; set; }
    }
}
